scenes/levels/LevelGenerator.gd

```diff
@@ -0,0 +1,52 @@
+extends Node2D
+
+export(String, DIR) var level_dir
+export(int) var level_depth
+export(int, 0, 3) var repeats
+
+var level_height = ProjectSettings.get_setting("display/window/size/height")
```

Comment:
This is tightly coupling level size to viewport size. If we plan to have levels with different dimentions, then we should get these values from the level scenes themselves after loading them.